quantum/keymap_extras/keymap_spanish_latinamerica.h

```diff
@@ -0,0 +1,154 @@
+#define ES_LCBR KC_QUOT // {
+#define ES_RCBR KC_NUHS // }
+// Row 4
+#define ES_LABK KC_NUBS // <
+#define ES_Z    KC_Z    // Z
+#define ES_X    KC_X    // X
+#define ES_C    KC_C    // C
+#define ES_V    KC_V    // V
+#define ES_B    KC_B    // B
+#define ES_N    KC_N    // N
+#define ES_M    KC_M    // M
+#define ES_COMM KC_COMM // ,
+#define ES_DOT  KC_DOT  // .
+#define ES_MINS KC_SLSH // -
+
+/* Shifted symbols
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───────┐
+ * │ º │ ! │ " │ # │ $ │ % │ & │ / │ ( │ ) │ = │ ? │ ¡ │       │
+ * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┤
+ * │     │   │   │   │   │   │   │   │   │   │   │ ¨ │ * │     │
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    │
+ * │      │   │   │   │   │   │   │   │   │   │   │ [ │ ] │    │
+ * ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴────┤
+ * │    │ > │   │   │   │   │   │   │   │ ; │ : │ _ │          │
+ * ├────┼───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬────┤
+ * │    │    │    │                        │    │    │    │    │
+ * └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘
+ */
+// Row 1
+#define ES_MORD S(ES_PIPE) // º
+#define ES_EXLM S(ES_1)    // !
+#define ES_DQUO S(ES_2)    // "
+#define ES_HASH S(ES_3)    // #
+#define ES_DLR  S(ES_4)    // $
+#define ES_PERC S(ES_5)    // %
+#define ES_AMPR S(ES_6)    // &
+#define ES_SLSH S(ES_7)    // /
+#define ES_LPRN S(ES_8)    // (
+#define ES_RPRN S(ES_9)    // )
+#define ES_EQL  S(ES_0)    // =
+#define ES_QUES S(ES_QUOT) // ?
+#define ES_IEXL S(ES_IQUE) // ¿
+// Row 2
+#define ES_DIAE S(ES_ACUT) // ¨ (dead)
+#define ES_ASTR S(ES_PLUS) // *
+// Row 3
+#define ES_LBRC S(ES_LCBR) // [
+#define ES_RBRC S(ES_RCBR) // ]
+// Row 4
+#define ES_RABK S(ES_LABK) // >
+#define ES_SCLN S(ES_COMM) // ;
+#define ES_COLN S(ES_DOT)  // :
+#define ES_UNDS S(ES_MINS) // _
+
+/* AltGr symbols
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───────┐
+ * │ ¬ │ ¡ │ “ │ £ │ ¢ │ ∞ │ § │ ¶ │ • │ ª │ º │ \ │ ≠ │       │
+ * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┤
+ * │     │ @ │ ∑ │ € │ ® │ † │ ¥ │ ¨ │ ı │ ø │ π │ « │ ~ │     │
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    │
+ * │      │   │ ß │ ∂ │ ƒ │ © │   │   │   │ λ │ ~ │ ^ │ ` │    │
+ * ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴────┤
+ * │    │ ≤ │   │ ≈ │ ç │ √ │ ∫ │ ~ │ µ │   │ … │ – │          │
+ * ├────┼───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬────┤
+ * │    │    │    │                        │    │    │    │    │
+ * └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘
+ */
+// Row 1
+#define ES_NOT  ALGR(ES_PIPE) // ¬
+#define ES_BSLS ALGR(ES_QUOT) // (backslash)
+// Row 2
+#define ES_AT   ALGR(ES_Q)    // @
+#define ES_EURO ALGR(ES_E)    // €
+
+// Row 3
+#define ES_TILD ALGR(ES_NTIL) // ~
+#define ES_CIRC ALGR(ES_LCBR) // ^ (dead)
+#define ES_GRV  ALGR(ES_RCBR) // ` (dead)
+
+//Row
+#define ES_CCED ALGR(ES_C)    // ç
```

Comment:
```suggestion
// Row 1
#define ES_NOT  ALGR(ES_PIPE) // ¬
#define ES_IEXL ALGR(ES_1)    // ¡
#define ES_LDQU ALGR(ES_2)    // “
#define ES_PND  ALGR(ES_3)    // £
#define ES_CENT ALGR(ES_4)    // ¢
#define ES_INFN ALGR(ES_5)    // ∞
#define ES_SECT ALGR(ES_6)    // §
#define ES_PILC ALGR(ES_7)    // ¶
#define ES_BULT ALGR(ES_8)    // •
#define ES_FORD ALGR(ES_9)    // ª
#define ES_MORD ALGR(ES_0)    // º
#define ES_BSLS ALGR(ES_QUOT) // (backslash)
#define ES_NEQL ALGR(ES_IQUE) // ≠

// Row 2
#define ES_AT   ALGR(ES_Q)    // @
#define ES_NARS ALGR(ES_W)    // ∑
#define ES_EURO ALGR(ES_E)    // €
#define ES_REGD ALGR(ES_R)    // ®
#define ES_DAGG ALGR(ES_T)    // †
#define ES_YEN  ALGR(ES_Y)    // ¥
#define ES_DIAE ALGR(ES_U)    // ¨ (dead)
#define ES_DLSI ALGR(ES_I)    // ı
#define ES_OSTR ALGR(ES_O)    // ø
#define ES_PI   ALGR(ES_P)    // π
#define ES_LDAQ ALGR(ES_ACUT) // «
#define ES_TILD ALGR(ES_PLUS) // ~

// Row 3
#define ES_SS   ALGR(ES_S)    // ß
#define ES_PDIF ALGR(ES_D)    // ∂
#define ES_FHK  ALGR(ES_F)    // ƒ
#define ES_COPY ALGR(ES_G)    // ©
#define ES_LAMB ALGR(ES_L)    // λ
#define ES_TILD ALGR(ES_NTIL) // ~
#define ES_CIRC ALGR(ES_LCBR) // ^
#define ES_GRV  ALGR(ES_RCBR) // `

// Row 4
#define ES_LTEQ ALGR(ES_LABK) // ≤
#define ES_AEQL ALGR(ES_X)    // ≈
#define ES_CCED ALGR(ES_C)    // ç
#define ES_SQRT ALGR(ES_V)    // √
#define ES_INTG ALGR(ES_B)    // ∫
#define ES_TILD ALGR(ES_N)    // ~
#define ES_MICR ALGR(ES_M)    // µ
#define ES_ELLP ALGR(ES_DOT)  // …
#define ES_NDSH ALGR(ES_MINS) // –
```

Reply:
This is based on a Mac(ISO) and I think and there are differences with other OSs. Mostly in the AltGr layer (I actually don't know if they behave the same in Windows and Linux). I will test it later to see the differences. 

About the symbols missing I didn't know what's to do...  because of the differences on the AltGr layer and because I only added only added the ones that I saw  in https://github.com/qmk/qmk_firmware/blob/master/docs/keycodes.md
or in the previous Spanish Keymap...    I didn't know how to name them, and had doubts if that would be good for non Mac users...  Thank you

Considering the differences between OS's maybe this should be tagged as macOS and I can make later a pull request for a 'regular one'...  but you know better what to do....

About the ES_TILD, probably the one in the third row because that's where the ñ letter (to the right of the L in US layout) is and it is and it is a bit easier to remember for a Spanish speaker..  it is weird that it is also defined with ALTGr+n

Reply:
If it is based on Mac, then `A(kc)` would be more appropriate than `ALGR(kc)`.

Reply:
If there are multiple ways to input the same character, the convention I've gone with is to keep only the first one that appears from least to most modified, top left to bottom right.